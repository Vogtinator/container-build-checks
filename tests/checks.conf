[General]
Vendor=org.opensuse
Registry=registry.opensuse.org
