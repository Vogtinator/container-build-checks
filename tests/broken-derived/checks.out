Looking at broken-derived.containerinfo
Warning: Using manually defined repositories (not obsrepositories:/) in the image. Only obsrepositories:/ is allowed.
Release: 7.21
Tag: example:latest
Warning: Tag example:latest is not namespaced (e.g. opensuse/foo)
Tag: opensuse/example:5.7
Tag: opensuse/example:5.7.7.21
Detected image specific label prefix: com.opensuse.example
Warning: Label prefix doesn't start with org.opensuse
Warning: Labels org.openbuildservice.disturl and com.opensuse.example.disturl not specified by this image
Warning: Labels org.opencontainers.image.title and com.opensuse.example.title not specified by this image
Warning: Label com.opensuse.example.description not identical to org.opencontainers.image.description
Warning: Labels org.opencontainers.image.version and com.opensuse.example.version not specified by this image
Warning: The org.opensuse.reference label (registry.opensuse.org/opensuse/example:5.7) does not refer to a tag identifying a specific build

container-build-checks done. Hints: 0 Warnings: 8 Errors: 0
Warnings found, but those are only fatal in certain projects.
Exited with 0
